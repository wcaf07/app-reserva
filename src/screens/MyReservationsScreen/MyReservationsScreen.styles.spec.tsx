import React from 'react';
import {render} from '@testing-library/react-native';
import {
  Container,
  TextItem,
  TextSubtitle,
  ItemList,
  TextSubValue,
} from './MyReservationsScreen.styles';

describe('MyReservationsScreen.styles', () => {
  test('THEN TextItem SHOULD match with snapshot when is show', () => {
    const component = render(<TextItem />).toJSON();

    expect(component).toMatchSnapshot();
  });
  test('THEN Container SHOULD match with snapshot when is show', () => {
    const component = render(<Container />).toJSON();

    expect(component).toMatchSnapshot();
  });
  test('THEN TextSubtitle SHOULD match with snapshot when is show', () => {
    const component = render(<TextSubtitle />).toJSON();

    expect(component).toMatchSnapshot();
  });
  test('THEN ItemList SHOULD match with snapshot when is show', () => {
    const component = render(<ItemList />).toJSON();

    expect(component).toMatchSnapshot();
  });
  test('THEN TextSubValue SHOULD match with snapshot when is show', () => {
    const component = render(<TextSubValue />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
