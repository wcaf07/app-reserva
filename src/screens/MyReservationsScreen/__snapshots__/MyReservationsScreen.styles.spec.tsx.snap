// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyReservationsScreen.styles THEN Container SHOULD match with snapshot when is show 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
      },
    ]
  }
/>
`;

exports[`MyReservationsScreen.styles THEN ItemList SHOULD match with snapshot when is show 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
      "paddingBottom": 5,
      "paddingLeft": 5,
      "paddingRight": 5,
      "paddingTop": 5,
    }
  }
/>
`;

exports[`MyReservationsScreen.styles THEN TextItem SHOULD match with snapshot when is show 1`] = `
<Text
  style={
    Array [
      Object {
        "color": "#000000",
        "fontSize": 15,
        "fontWeight": "bold",
      },
    ]
  }
/>
`;

exports[`MyReservationsScreen.styles THEN TextSubValue SHOULD match with snapshot when is show 1`] = `
<Text
  style={
    Array [
      Object {
        "fontSize": 13,
      },
    ]
  }
/>
`;

exports[`MyReservationsScreen.styles THEN TextSubtitle SHOULD match with snapshot when is show 1`] = `
<Text
  style={
    Array [
      Object {
        "fontSize": 10,
      },
    ]
  }
/>
`;
